frontend/testing/playwright/tests/dashboard/dashboard.teardown.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from '@playwright/test';
+import { Gitea } from '../../helpers/Gitea';
+import { test } from '../../extenders/testExtend';
+
+test('Delete the test app', async ({ request, testAppName }) => {
+  const gitea = new Gitea();
+  const response = await request.delete(gitea.getDeleteAppEndpoint({ app: testAppName }));
+  expect(response.ok()).toBeTruthy();
+
+  const response2 = await request.delete(gitea.getDeleteAppEndpoint({ app: `${testAppName}2` }));
+  expect(response2.ok()).toBeTruthy();
```

Comment:
What do you think about doing something like following to avoid repeating the code implementation for deleting the app?

```suggestion
  const gitea = new Gitea(); 
  const appsToDelete: string[] = [testAppName, `${testAppName}2`];
  
  for (const app of appsToDelete) {
      const response = await request.delete(gitea.getDeleteAppEndpoint({ app }));
      expect(response.ok()).toBeTruthy();
  }
```